### PR TITLE
bash: refresh page: add here-doc example and parameter expansion examples

### LINKED ISSUE #95: [Request] Heredoc and parameter expansion
Hi Cheat developer,

Shall we add [Heredoc](https://www.xaprb.com/media/2007/03/bash-parameter-expansion-cheatsheet.pdf) and [Parameter](https://linuxize.com/post/bash-heredoc/) expansion to bash cheat please?

Thanks and regards,
Casper

### PR DESCRIPTION
**Pull request by EmilyGraceSeville7cf**
closes #95

### CODE REVIEW

File: bash
Comment (edgester) [inline bash]: I recommend removing the indent. I understand that it improves readability, but it can be confusing to users, because the indent will become part of the contents.

**Review by edgester — CHANGES_REQUESTED** [review]
bash variables use curly braces, not angle brackets.  For example `echo ${variable}`

**Comment by edgester** [comment]
Fixes issue #95

**Comment by EmilyGraceSeville7cf** [comment]
> bash variables use curly braces, not angle brackets. For example echo ${variable}

Yeah, but it's not a bash syntax, it's placeholder syntax that used over bash syntax. :) I suggested to stick to shell syntax only to @chrisallenlane, but he declined this idea.

**Comment by edgester** [comment]
@EmilySeville7cfg Thanks for clarifying. I think the placeholder syntax is confusing in this case, but @chrisallenlane has the final word.

**Comment by EmilyGraceSeville7cf** [comment]
I am closing this PR to make project maintainer pay attention for it.

**Pull request closed without merging**
